### PR TITLE
added cart-management page

### PR DESCRIPTION
**Pull request by vidhi-thakur**
added a cart management page with carts and a table to display total amount
image

check preview URL [here](https://deploy-preview-5--savvy-store.netlify.app/pages/cart-management.html)

### CODE REVIEW

**Review by shuklahimanshu81 — APPROVED** [review]
Everytime I review your code I learn something new. Code structure is super clean. Everything is good. Go for merge.

**Review by bharati-21 — APPROVED** [review]
Looks good to be merged. Great use of variables! 🚀

**Comment by vidhi-thakur** [comment]
> Looks good to be merged. Great use of variables! 🚀

thank you

**Review by ajit-sharmaGH — APPROVED** [review]
looks good

File: styles/variable.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+html {
+  font-family: var(--default-font-family);
+  box-sizing: border-box;
+  font-size: var(--default-font-size);
+}
+
+body {
+  min-height: 100vh;
+  background-color: #f7f7f7;
```
Comment (ajit-sharmaGH) [inline styles/variable.css:89]: you can use css variables

**Review by ajit-sharmaGH — APPROVED** [review]

**Pull request merged**